Library/Homebrew/extend/os/linux/requirements/osxfuse_requirement.rb

```diff
@@ -0,0 +1,37 @@
+require "requirement"
+
+class OsxfuseRequirement < Requirement
+  download "https://github.com/libfuse/libfuse"
```

Comment:
If this doesn't 💥 if this formula is missing (i.e. try it with `libfuse2`) I'm fine for the `default_formula` to be set.